src/views/DefineCompany.vue

```diff
@@ -11,9 +11,11 @@
       <header>
         <h2>1. Company Name</h2>
       </header>
-      <v-card flat class="step-container">
-        <name-request-info @hasNameTranslation="onNameTranslation($event)"/>
-      </v-card>
+      <div :class="{ 'invalid-section': showErrors && !hasValidNameTranslation }">
```

Comment:
Just curious: would setting the "invalid-section" class on the v-card work?

Reply:
I tried and v-card has a white border in light theme